src/ert/run_models/everest_run_model.py

```diff
@@ -733,7 +733,7 @@ def _get_run_args(
         substitutions = self.substitutions
         self.active_realizations = [True] * len(sim_to_model_realization)
         for sim_id, model_realization in enumerate(sim_to_model_realization):
-            substitutions[f"<GEO_ID_{sim_id}_{ensemble.iteration}>"] = str(
+            substitutions[f"<MODEL_ID_{sim_id}_{ensemble.iteration}>"] = str(
```

Comment:
This looks like a breaking change to users config?

We need to allow GEO_ID still, but issue a deprecation warning if used.

Reply:
Not related to this part of the code. The tree structure we write to disk is also changed. IE:
```

├── sim_output
│   ├── batch_0
│   │   └── realization_0 (from geo_realization_0)
│   │       ├── simulation_0
```
 
How should we handle this? add migrations